### PR TITLE
Rename near-bindgen to near-sdk

### PR DESCRIPTION
**Pull request by nearmax**

### CODE REVIEW

File: near-sdk-core/Cargo.toml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 name = "near-sdk-core"
-version = "0.6.0"
+version = "0.6.1"
 authors = ["Near Inc <max@nearprotocol.com>"]
 edition = "2018"
 license = "Apache-2.0"
 categories = ["wasm"]
-repository = "https://github.com/nearprotocol/near-sdk"
-homepage = "https://github.com/nearprotocol/near-sdk"
+repository = "https://github.com/nearprotocol/near-sdk-rs"
```
Comment (frol) [inline near-sdk-core/Cargo.toml]: Should it be `near/near-sdk-rs`?

Comment (nearmax) [reply near-sdk-core/Cargo.toml]: Fixed

File: README.md [hunk 2]
```diff
@@ -1,40 +1,44 @@
...
+    <a href="https://crates.io/crates/near-sdk"><img src="https://img.shields.io/crates/v/near-sdk.svg?style=flat-square" alt="Crates.io version" /></a>
+    <a href="https://crates.io/crates/near-sdk"><img src="https://img.shields.io/crates/d/near-sdk.svg?style=flat-square" alt="Download" /></a>
     <a href="https://discord.gg/gBtUFKR"><img src="https://img.shields.io/discord/490367152054992913.svg" alt="Join the community on Discord" /></a>
-    <a href="https://travis-ci.com/nearprotocol/near-bindgen"><img src="https://travis-ci.com/nearprotocol/near-bindgen.svg?branch=master" alt="Travis Build" /></a>
+    <a href="https://travis-ci.com/nearprotocol/near-sdk"><img src="https://travis-ci.com/nearprotocol/near-sdk.svg?branch=master" alt="Travis Build" /></a>
   </p>
 
    <h3>
-      <a href="https://github.com/nearprotocol/near-bindgen#features">Features</a>
+      <a href="https://github.com/nearprotocol/near-sdk#features">Features</a>
```
Comment (frol) [inline README.md]: Should we just make these links relative? `#features` vs `https://github.com/nearprotocol/near-sdk#features`?

File: README.md [hunk 3]
```diff
@@ -1,40 +1,44 @@
...
+
 
   <p>
-    <a href="https://crates.io/crates/near-bindgen"><img src="https://img.shields.io/crates/v/near-bindgen.svg?style=flat-square" alt="Crates.io version" /></a>
-    <a href="https://crates.io/crates/near-bindgen"><img src="https://img.shields.io/crates/d/near-bindgen.svg?style=flat-square" alt="Download" /></a>
+    <a href="https://crates.io/crates/near-sdk"><img src="https://img.shields.io/crates/v/near-sdk.svg?style=flat-square" alt="Crates.io version" /></a>
+    <a href="https://crates.io/crates/near-sdk"><img src="https://img.shields.io/crates/d/near-sdk.svg?style=flat-square" alt="Download" /></a>
     <a href="https://discord.gg/gBtUFKR"><img src="https://img.shields.io/discord/490367152054992913.svg" alt="Join the community on Discord" /></a>
-    <a href="https://travis-ci.com/nearprotocol/near-bindgen"><img src="https://travis-ci.com/nearprotocol/near-bindgen.svg?branch=master" alt="Travis Build" /></a>
+    <a href="https://travis-ci.com/nearprotocol/near-sdk"><img src="https://travis-ci.com/nearprotocol/near-sdk.svg?branch=master" alt="Travis Build" /></a>
```
Comment (frol) [inline README.md]: This also needs to be fixed.

File: README.md [hunk 4]
```diff
@@ -75,14 +79,14 @@ impl StatusMessage {
     * `promise_and` -- combinator, allows waiting on several promises simultaneously, before executing the callback;
     * `promise_return` -- treats the result of execution of the promise as the result of the current function.
     
-    Follow [examples/cross-contract-high-level](https://github.com/nearprotocol/near-bindgen/tree/master/examples/cross-contract-high-level)
+    Follow [examples/cross-contract-high-level](https://github.com/nearprotocol/near-sdk/tree/master/examples/cross-contract-high-level)
```
Comment (frol) [inline README.md]: Let's search for `github.com/nearprotocol` and update them all :smile:

Comment (nearmax) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (frol) [reply README.md]: @nearmax Is this Travis link is left pointing to nearprotocol on purpose?

Comment (nearmax) [reply README.md]: Nope, a mistake. I asked @ailisp to give me instruction on how to fix CI now that I have transferred it, I will fix it as a part of that.

**Comment by evgenykuzyakov** [comment]
The version incorrect. It should be `0.7.0` since it's a breaking API change.

**Comment by nearmax** [comment]
> The version incorrect. It should be `0.7.0` since it's a breaking API change.

Yeah, technically it would've been the right thing to do, but I treated `near-sdk` as a new package rather than an evolution of `near-bindgen`. Since the name of the package changed we won't be breaking anyone using the old package.

**Pull request merged**